planner/core/logical_plan_builder.go

```diff
@@ -2975,10 +2975,16 @@ func (b *PlanBuilder) buildUpdateLists(
 ) (newList []*expression.Assignment,
 	po LogicalPlan,
 	allAssignmentsAreConstant bool,
-	error error,
+	e error,
```

Comment:
why name it as `e` here?

Reply:
because it will conflict with https://github.com/pingcap/tidb/pull/13530/files#diff-b9e75a99170ea35a67074d5d3d3305adR3039